### DISCUSSION TITLE
Angular Editor (Quill) documentation missing angular.json styles entry

### DISCUSSION THREAD

**Discussion by cheezist** [primefaces/community#385] [PrimeNG] [answered]
Hello, I was trying to implement the <p-editor> today and got some really weird artifacts on my screen. I saw this occur on all of the StackBlitz examples on the documentation page as well: https://primeng.org/editor#quill

The fix was (for me) was to include the "./node_modules/quill/dist/quill.snow.css" entry in the "styles" section of my angular.json file. Here is a StackBlitz example I changed to make the example work as expected: https://stackblitz.com/edit/ctebfj-os8cvk?file=angular.json

**Comment by cetincakiroglu** [comment]
Hi,

Could you please explain what is the issue, what were you implementing?
Issue is not clear.

**Reply by cheezist** [reply]
Sure, I was simply trying to use the `<p-editor />` component in my app. Unfortunately following the documentation did not result in a working editor. The documentation needs to be updated to state that one of the Quill theme files needs to be included in angular.json "styles" section to work correctly, otherwise the editor is useless and shows odd artifacts on the screen. Look at the StackBlitz example for this example and see what I'm talking about for the latest 16.7.2: [https://primeng.org/editor#basic](https://primeng.org/editor#basic)

Once the "styles" entry in angular.json has been updated to include a Quill theme file, in my case it was "./node_modules/quill/dist/quill.snow.css", then the editor started to show in the app as expected. The link I posted was a fork of the basic editor example with the angular.json file updated.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Sorry for the late response and thanks a lot for reporting the issue, we'll update the documentation for it!
